### PR TITLE
data: akshitagupta15june

### PR DESCRIPTION
**Pull request by akshitagupta15june**
## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [x] My change requires changes to the documentation.
- [x] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/4831"></a>

### CODE REVIEW

File: data/akshitagupta15june.json [hunk 1]
```diff
@@ -0,0 +1,32 @@
+{
+  "name": "Akshita Gupta",
+  "type": "personal",
+  "bio": "I love Hackathons, open source and sports"
```
Comment (swagataroy30) [inline data/akshitagupta15june.json:4]:
```suggestion
  "bio": "I love Hackathons, open source and sports",
```

**Review by swagataroy30 — COMMENTED** [review]
You need to add a comma at line 4. Thank You!

**Comment by akshitagupta15june** [comment]
Done @eddiejaoude

**Comment by akshitagupta15june** [comment]
> You need to add a comma at line 4. Thank You
Done

**Review by Pradumnasaraf — APPROVED** [review]
Looks good to merge. Welcome to LinkFree 🎉

**Pull request merged**